sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1620,6 +1621,35 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
       spval,VarName,SFCEXC)
 
+! accumulated snowfall
+      VarName='tsnowp'
```

Comment:
@EricJames-NOAA You will submit a PR to output some GSL precip type variables for RRFS.  I would like to know if there are any conflicts since 3 GSL variables (acgraup, graup_bucket, acfrain) were assigned to read new winter diag fields from model.